stablehlo/reference/Process.h

```diff
@@ -39,6 +39,9 @@ class Process {
   ProcessGroups crossPartition(
       SmallVector<SmallVector<uint32_t>> partitionGroups);
 
+  /// See `ProcessGrid::outfeed`.
+  void outfeed(ArrayRef<Tensor> inputs);
```

Comment:
Let's order methods alphabetically? This will make it easier to decide whether to put new methods.

Reply:
I see that methods are now ordered in ProcessGrid.h, but I don't think they are in this file.